subjects/printstr/README.md

```diff
@@ -30,8 +30,7 @@ And its output :
 
 ```console
 $ go run . | cat -e
-Hello World!
-$
+Hello World!$
```

Comment:
This is tricky to fix at the moment. This is valid, but also misleading. The `$` represents both the prompt and `cat -e` new-line position. So it is difficult to understand which is being used here. See https://github.com/01-edu/public/issues/1304.

One thing we could try here is to represent the prompt and new-line position with different characters. Perhaps the prompt could be something like `prompt>` or something else appropriate.

Reply:
I see what you mean, it makes sense. Wouldn't it be less confusing then, to remove the $ at the beginning of the command, since different prompts represent it in different ways, and its the source of the problem? Should I remove it as well?

We are specifying that it's the output, so wouldn't it be creating less confusion for the students and thereby less issues to solve? Perhaps modify line 29  then to say `And its output to the terminal:`

I think that even if not ideal it would be better. What do you think?

Reply:
To be more specific, what I proposed is something like this:
`And its output to the the terminal:`
- without new line required:
```console
go run . | cat -e
Hello World!
```
- with new line required: 
```console
go run . | cat -e
Hello World!$
```

Reply:
It's quite common in console type code-blocks, that commands are distinct from output. I guess we don't actually _need_ to show the command though in this case. I think it was used in this case to demonstrate that the function should print a string which is not new-line terminated (using `cat -e`).

```console
bash-3.2$ echo -n "frogs are cool"
frogs are coolbash-3.2$
```

```console
bash-3.2$ echo -n "frogs are cool" | cat -e
frogs are coolbash-3.2$
```

```console
bash-3.2$ echo "frogs are cool" | cat -e
frogs are cool$
bash-3.2$
```

In the long term, the intention is to make the prompt distinct. But in the short term, you can do whatever makes the exercise more clear.

---

You can implement any fix which makes the subject clearer.

Reply:
Done

Reply:
I am resolving because the pull request was approved.